misc/test_site/mac_addrs/9a02571e8f01/module_config.json

```diff
@@ -24,13 +24,29 @@
   "servers": {
     "tcp": {
       "ports": {
-        "443": {
-          "allowed": true,
-          "reason": "Administrative Web-Server"
+        "23": {
+          "allowed": false,
+          "description": "Telnet Server",
+          "reason": ""
```

Comment:
nb: empty reason is not necessary -- it would only be required if there was a reason that the port was open..

Reply:
Sorry about the junk, yes I think it should have been rebased. I'll fix it later this evening. 
On the empty reason, I thought that providing a template field in the file in the qualification folder was good. Shall I keep it in the qualification folder and remove from the device configuration inside misc?